kitsune/settings.py

```diff
@@ -806,13 +796,10 @@ def JINJA_CONFIG():
 
 #
 # Sessions
-SESSION_COOKIE_AGE = config('SESSION_COOKIE_AGE', default=4 * 7 * 24 * 60 * 60, cast=int)  # 4 weeks
 SESSION_COOKIE_SECURE = config('SESSION_COOKIE_SECURE', default=not DEBUG, cast=bool)
 SESSION_COOKIE_HTTPONLY = True
-SESSION_EXPIRE_AT_BROWSER_CLOSE = False
 SESSION_COOKIE_NAME = 'session_id'
 SESSION_ENGINE = config('SESSION_ENGINE', default='django.contrib.sessions.backends.cache')
-SESSION_EXISTS_COOKIE = 'sumo_session'
```

Comment:
There's two more references to `SESSION_EXISTS_COOKIE` which need to be removed:

kitsune/users/middleware.py L24
kitsune/users/views.py L84